servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/ConcurrentAsyncContextMap.java

```diff
@@ -15,30 +15,37 @@
  */
 package io.servicetalk.concurrent.api;
 
+import io.servicetalk.concurrent.internal.ContextMapUtils;
+import io.servicetalk.context.api.ContextMap;
+
 import java.util.Map;
 import java.util.Map.Entry;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
 import java.util.function.BiPredicate;
+import java.util.function.Function;
 import javax.annotation.Nullable;
 
-import static io.servicetalk.concurrent.api.AsyncContextMapUtils.contextMapToString;
+final class ConcurrentAsyncContextMap implements ContextMap {
```

Comment:
It doesn't always work but one way to work around this is to branch off of main, `git mv` the file(s) to their new names but make no other changes, commit and and then rebase on top of that change. Sometimes this will convince it to track the name change and other times it will still just forget.

Reply:
Correct, `git mv` helps when you have two independent commits/PRs. Inside one PR (and after "squash & merge") it will be again shown as a new file. I tried to avoid that.

Reply:
Renamed in #1959